src/challenges.js

```diff
@@ -19,11 +29,24 @@ const counter = () => {
   // Example: const newCounter = counter();
   // newCounter(); // 1
   // newCounter(); // 2
+  let count = 0;
+  return () => {
+    count++;
+    return count;
```

Comment:
Another way you can refactor this code is having it as 
`return () => {
  return ++count;
}`
This is just from that one time of the difference between count++ and ++count. But either way, your work is correct!